scrunch/datasets.py

```diff
@@ -1398,7 +1398,50 @@ def create_categorical(self, categories, alias, name, multiple, description='',
 
         Will create either categorical variables or multiple response depending
         on the `multiple` parameter.
+
+        If the variable to create is a Multiple Reponse with the 3 default categories:
+            1: Selected
+            2: Not Selected
+            3: No Data
+        then the categories argument must include a key 'missing_case' and must declate subvariables
+        for the 3 categories, i.e.:
+            categories: [
+                {
+                    'case': 'nonexistant == 1',
+                    'name': 'subvar_1',
+                    'id': 1,
+                    'missing_case': 'nonexistant == 3'
+                },
+                {
+                    'case': 'nonexistant == 2',
+                    'name': 'subvar_2',
+                    'id': 2,
+                    'missing_case': 'nonexistant == 4'
+                },
+                {
+                    'case': 'nonexistant == 3',
+                    'name': 'subvar_3',
+                    'id': 3,
+                    'missing_case': 'nonexistant == 5'
+                }]
         """
+        missing_case = 'missing_case' in [c.keys() for c in categories]
+        if multiple and missing_case:
+            # This is a special case to create a 3 categories MR
+            _categories = [
+                {'id': 1, 'name': 'Selected', 'selected': True},
+                {'id': 2, 'name': 'Not Selected'},
+                {'id': 3, 'name': 'No Data', 'missing': True}
+            ]
+            _subvariables = []
+            for sv in categories:
+                _subvariables.append(
+                    {'id': sv['id'], 'name': sv['name'], 'cases': {1: sv['case'], 2: sv['missing_case']}}
+                )
```

Comment:
This is currently mapping `missing_case` to `not selected` but it should map to `No Data`.

Reply:
yes, this is a bug.